recipes/alsa/all/conanfile.py

```diff
@@ -0,0 +1,120 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+
+class AlsaConan(ConanFile):
+    name = "alsa"
```

Comment:
Why not add this functionality to `libalsa`?
Having 2 recipes will lead to duplicated work.

Reply:
@madebr Yeah will change back. I was thinking we would rename libalsa to alsa since now it also includes the plug-in and is not just the library but now I think it is better to keep the name the same.